core/arch/aarch64/aarch64.asm

```diff
@@ -1,5 +1,5 @@
 /* **********************************************************
```

Comment:
> to that PR the FP regs were simply stored in a continuous manner in signal handling code, and fcache enter/exit routines 

s/continuous/contiguous/
s/code, and/code and/

Reply:
Done.